mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerator
+	METHOD m_afzjxjzq nextInt (I)I
+		ARG 1 bound
+	METHOD m_bggirwfo createPositionalRandomFactory ()Lnet/minecraft/unmapped/C_coqzjjrt;
+	METHOD m_culopwlx skip (I)V
+		ARG 1 count
+	METHOD m_djbqzhqs nextFloat ()F
+	METHOD m_ezprnxnd nextInt ()I
+	METHOD m_fakwfybt nextDouble ()D
+	METHOD m_ifdmmgky nextLong ()J
+	METHOD m_imwpekuz createUnsafe ()Lnet/minecraft/unmapped/C_rlomrsco;
+	METHOD m_iwfdczgh setSeed (J)V
+		ARG 1 seed
+	METHOD m_lzlwtgyn nextGaussian ()D
+	METHOD m_mkdxraps nextBoolean ()Z
+	METHOD m_ngqtmirh createLegacy (J)Lnet/minecraft/unmapped/C_rlomrsco;
+		ARG 0 seed
+	METHOD m_paekwdby derive ()Lnet/minecraft/unmapped/C_rlomrsco;
+	METHOD m_schtxhxl range (II)I
+		ARG 1 min
+		ARG 2 max
+	METHOD m_sgbekski nextBetween (II)I
```

Comment:
I'm assuming this is a good mapping name, but just to ask anyone who feels like explaining: what's the difference between `range` and `nextBetween`?  These names, without any personal context with the code, mean to me that range will constrain random values to a certain range, while nextBetween would find the first random value that fits in a certain range.  What does the distinction between range and between signify?  Is it just to make it clear they are separate methods?

Reply:
After checking the code the second should be `inclusiveRange`